web-application/src/certificates/certificate-creation.component.ts

```diff
@@ -21,7 +21,7 @@ export class CertificateCreationComponent {
     delete = false;
 
     jsonThing = JSON.stringify;
-    states = [{type: 'ISSUED'}, {type: 'REVOKED'}];
+    states = [{type: 'ISSUED'}, {type: 'REVOKED'}, {}];
```

Comment:
What does this empty object signify? Is it for deleted certificates?

Reply:
Should have been for EXPIRED states whoops :)